ISSUE_TEMPLATE.md

```diff
@@ -1,18 +1,17 @@
-If you are reporting a bug or problem with dlib then fill out the entire template below.  Issues asking questions about why something doesn't work that fail to fill out the entire template will be closed.
+IF YOU ARE REPORTING A BUG OR PROBLEM WITH DLIB THEN FILL OUT THE ENTIRE TEMPLATE BELOW.  ISSUES ASKING QUESTIONS ABOUT WHY SOMETHING DOESN'T WORK THAT FAIL TO FILL OUT THE ENTIRE TEMPLATE WILL BE CLOSED.
```

Comment:
Why this file changed?

Reply:
I'm not sure.  I guess Davis capitalized it for emphasis.  I think people don't usually fill out the issue.  Or ... is that not what you are asking?

Reply:
I think patch was strarted from wrong base branch or some unrelated changes leaked. I expect tiny change like this one: https://github.com/hunter-packages/dlib/pull/12/files

Reply:
This was based on the latest 19.8.X release, since it includes some OS X related fixes that will be needed for CUDA.

https://github.com/headupinclouds/dlib/blob/33ec39d0548a80fa23f1333c60f32b90a5abf9d6/dlib/CMakeLists.txt#L22-L24
```
set(CPACK_PACKAGE_VERSION_MAJOR "19")
set(CPACK_PACKAGE_VERSION_MINOR "8")
set(CPACK_PACKAGE_VERSION_PATCH "99")
```

Reply:
It's a part of the commit. It will be hard to cherry-pick it later. I'll create a new branch.